### ISSUE TITLE
store.configureStore is not a function when doing unit test

### ISSUE THREAD

**Issue by justin73** [angular-redux/store#419]
### This is a...

* [ ] feature request
* [x] bug report
* [x] usage question

### What toolchain are you using for transpilation/bundling?

* [x] @angular/cli
* [ ] Custom @ngTools/webpack
* [ ] Raw `ngc`
* [ ] SystemJS
* [ ] Rollup
* [ ] Other

### Environment

NodeJS Version: 7.10.0
Typescript Version: 2.3.2
Angular Version:4.1.0
@angular-redux/store version: 6.4.2
@angular/cli version: (if applicable) 1.0.2
OS: Windows 10

### Link to repo showing the issues
(optional, but helps _a lot_)

### Expected Behaviour:
Basically, I am using the Zoo demo as the redux store prototype to setup my project. 

In my unit test for my project. I want to test the StoreModule.ts, so I created a StoreModule instance to test the initialization of the redux store. I should be able to get the instance with no error. 

### Actual Behaviour:

When I was at version 6.2.1, I tried to create the StoreModule instance and passed all the necessary parameters, everything works fine. The instance of the store got created.

But I updated to 6.4.3 and now when I tried to do the same thing, it failed. The error shows Store.configureStore is not a function. 

![error](https://cloud.githubusercontent.com/assets/3356603/26802685/72802992-4a0f-11e7-926e-f7f25c21b160.JPG)
![store module](https://cloud.githubusercontent.com/assets/3356603/26802689/7525493e-4a0f-11e7-8157-b494bcaca60a.JPG)

### Stack Trace/Error Message:

Failed: store.configureStore is not a function

### Additional Notes:
(optional)
When I ran my project, everything works fine. But the error only happens when I run the unit test.  I know that the fractal store got added to the lib, but that I don't know why the test is failing. So I am not sure if it is a usage question or an actual bug. 

Thank you!

**Comment by SethDavenport** [comment]
Can you clarify what you're passing in for `mockNgRedux`?

**Comment by justin73** [comment]
So here is the entire test suite to test the StoreModule. 
`describe('Module Reducer', () => {
  let backend: MockBackend;
  let epic: RootEpics;
  let devTools: DevToolsExtension;
  let mockNgRedux;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [NgReduxTestingModule, HttpModule],
      providers: [
        BaseRequestOptions,
        MockBackend,
        RootEpics,
        ModuleEpics,
        ModuleService,
        ModuleActions,
        DevToolsExtension,
        NgReduxRouter
      ]
    }).compileComponents().then(() => {
      const testbed = getTestBed();
      backend = testbed.get(MockBackend);
      epic = testbed.get(RootEpics);
      devTools = testbed.get(DevToolsExtension);
      mockNgRedux = MockNgRedux.mockInstance;
    });
  }));

  it('should initiate a StoreModule and create an instance', async(() => {
    const instance = new StoreModule(mockNgRedux, devTools, null, epic);
    expect(instance).toBeDefined();
  }));
});`

**Comment by justin73** [comment]
So here is the test suite. I set mockNgRedux as MockNgRedux.mockInstance. Didn't assign any specific value to it. 
```
describe('Module Reducer', () => {
  let backend: MockBackend;
  let epic: RootEpics;
  let devTools: DevToolsExtension;
  let mockNgRedux;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [NgReduxTestingModule, HttpModule],
      providers: [
        BaseRequestOptions,
        MockBackend,
        RootEpics,
        ModuleEpics,
        ModuleService,
        ModuleActions,
        DevToolsExtension,
        NgReduxRouter
      ]
    }).compileComponents().then(() => {
      const testbed = getTestBed();
      backend = testbed.get(MockBackend);
      epic = testbed.get(RootEpics);
      devTools = testbed.get(DevToolsExtension);
      mockNgRedux = MockNgRedux.mockInstance;
      console.log(mockNgRedux);
    });
  }));

  it('should initiate a StoreModule and create an instance', async(() => {
    const instance = new StoreModule(mockNgRedux, devTools, null, epic);
    expect(instance).toBeDefined();
  }));
});
```

**Comment by SethDavenport** [comment]
OK reproduced. I'll take a look at a fix.

I'm curious about this test though - is it really necessary to test that the constructor creates an instance?

**Comment by justin73** [comment]
Well, when it came to this store.module.ts file, it was nothing else but the constructor. I figured why not just test to see the instance is defined then gets a 100% coverage of this file. Probably it was not a must-do test case... but thanks for looking into it.

**Comment by SethDavenport** [comment]
Can you give `@angular-redux/store@6.4.5-alpha.0` a try?

**Comment by SethDavenport** [comment]
Can you confirm that `@angular-redux/store@6.4.5-alpha` works? If so, I'll release 6.4.5 for real. Thanks.

**Comment by justin73** [comment]
@SethDavenport Sorry for the delay. It works for me. I was just waiting for my teammate to confirm it as well. Thanks for the fix!

**Comment by SethDavenport** [comment]
Sweet. I'll release 6.4.5 later today.

**Comment by SethDavenport** [comment]
Fixed in 6.4.5.
